t/26-controllerrunning.t

```diff
@@ -70,7 +70,7 @@ subtest streaming => sub {
     my @fake_data = ("Foo bar\n", "Foo baz\n", "bar\n");
     my $tmpdir = path($controller->stash('job')->worker->{WORKER_TMPDIR});
     my $t_file = $tmpdir->child('test.txt');
-    $t_file->spurt(@fake_data);
+    $t_file->spew(@fake_data);
```

Comment:
That's wrong usage now, I think it needs to be
```suggestion
    $t_file->spew(join '', @fake_data);
```
```
sub spew {
  my ($self, $content, $encoding) = @_;
...
}
  
sub spurt {
  deprecated 'Mojo::File::spurt is deprecated in favor of Mojo::File::spew';
  shift->spew(join '', @_);
}
```

Reply:
Or can you pass an array ref?

Reply:
Nope: https://metacpan.org/pod/Mojo::File#spew
```
$path = $path->spew($bytes);
$path = $path->spew($chars, 'UTF-8');
```

Reply:
done